projects/assets-library/src/icons/icon-library.module.ts

```diff
@@ -51,6 +51,7 @@ const iconsRootPath = 'assets/icons';
         { key: IconType.Eye, url: `${iconsRootPath}/eye.svg` },
         { key: IconType.FileCode, url: `${iconsRootPath}/file-code.svg` },
         { key: IconType.Filter, url: `${iconsRootPath}/filter.svg` },
+        { key: IconType.FilterSpecial, url: `${iconsRootPath}/filter-special.svg` },
```

Comment:
When would we use this filter icon instead of the other one?

Reply:
We could rename the existing one to 'drilldown-filter'. That is the purpose of that icon in the UI now.